plugin/src/shared/cli-utils.ts

```diff
@@ -0,0 +1,24 @@
+import * as readline from 'readline';
+import * as fs from 'fs';
+
+export const askQuestion = (question: string): Promise<string> => {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  return new Promise((resolve) =>
+    rl.question(question, (answer) => {
+      rl.close();
+      resolve(answer);
+    })
+  );
+};
+
+export const getNxLibsPaths = () => {
```

Comment:
Let's make this util more flexible - it can accept array of library types:


```ts
enum LibraryType {
  UI = 'ui',
  DATA_ACCESS = 'data-access',
  FEATURES = 'features',
  UTILS = 'utils',
}

...
export const getNxLibsPaths = (types: Array<LibraryType>) => {
...
```

Reply:
done